test/pkg/environment/azure/environment.go

```diff
@@ -74,6 +76,7 @@ func NewEnvironment(t *testing.T) *Environment {
 	cred := lo.Must(azidentity.NewDefaultAzureCredential(nil))
 	azureEnv.VNETClient = lo.Must(armnetwork.NewVirtualNetworksClient(azureEnv.SubscriptionID, cred, nil))
 	azureEnv.InterfacesClient = lo.Must(armnetwork.NewInterfacesClient(azureEnv.SubscriptionID, cred, nil))
+	azureEnv.AKSManagedClusterClient = lo.Must(containerservice.NewManagedClustersClient(azureEnv.SubscriptionID, cred, nil))
```

Comment:
yay glad to see the cred stuff I setup being useful here.

Reply:
Yea, was super easy to expand on! Much appreciated :)